File: uiapplication/src/main/java/com/futing/uiapplication/fish/MainActivity.java
package com.futing.uiapplication.fish;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import com.futing.uiapplication.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_fish);
//        ImageView iv = findViewById(R.id.iv_fish);
//        iv.setImageDrawable(new FishDrawable());
    }
}

File: memorylib/src/main/java/com/futing/memorylib/MainActivity.kt
package com.futing.memorylib

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
//        enableEdgeToEdge()
        setContentView(R.layout.activity_main_memory)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

//        matTest()
//
//        Thread(){
//            kotlin.run {
//                Thread.sleep(1000*30)
//            }
//        val result = Utils.createDumpFile(this)

//        }.start()

    }

    private fun matTest() {
        TODO("Not yet implemented")

    }


    fun dumpClick(view: View) {
        incomingVsoutgoing()
        val result = Utils.createDumpFile(this)
        Utils.show(this, result)
    }

    private fun incomingVsoutgoing() {
        val a = A()
        val b = B()
    }
}